[2023-02-13 09:21:55,480] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 7_dataproc_subdags.par_task.avg_tincome scheduled__2023-02-12T00:00:00+00:00 [queued]>
[2023-02-13 09:21:55,487] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 7_dataproc_subdags.par_task.avg_tincome scheduled__2023-02-12T00:00:00+00:00 [queued]>
[2023-02-13 09:21:55,487] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-13 09:21:55,488] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-13 09:21:55,488] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-13 09:21:55,497] {taskinstance.py:1377} INFO - Executing <Task(DataprocSubmitJobOperator): avg_tincome> on 2023-02-12 00:00:00+00:00
[2023-02-13 09:21:55,499] {standard_task_runner.py:52} INFO - Started process 3176 to run task
[2023-02-13 09:21:55,502] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', '7_dataproc_subdags.par_task', 'avg_tincome', 'scheduled__2023-02-12T00:00:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/Airflow2_0_De_0_a_Heroe/7_dataproc_subdags.py', '--cfg-path', '/tmp/tmph69xzbhd', '--error-file', '/tmp/tmpbl6m_mxe']
[2023-02-13 09:21:55,503] {standard_task_runner.py:80} INFO - Job 83: Subtask avg_tincome
[2023-02-13 09:21:55,554] {task_command.py:369} INFO - Running <TaskInstance: 7_dataproc_subdags.par_task.avg_tincome scheduled__2023-02-12T00:00:00+00:00 [running]> on host Tinmar.
[2023-02-13 09:21:55,621] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=David Sanchez
AIRFLOW_CTX_DAG_ID=7_dataproc_subdags.par_task
AIRFLOW_CTX_TASK_ID=avg_tincome
AIRFLOW_CTX_EXECUTION_DATE=2023-02-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-12T00:00:00+00:00
[2023-02-13 09:21:55,625] {dataproc.py:1866} INFO - Submitting job
[2023-02-13 09:21:55,636] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-02-13 09:21:56,140] {dataproc.py:1878} INFO - Job b1b63d0a_subjob_avg_tincome submitted successfully.
[2023-02-13 09:21:56,160] {dataproc.py:1898} INFO - Waiting for job b1b63d0a_subjob_avg_tincome to complete
[2023-02-13 09:22:27,251] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/tinmar/.local/lib/python3.10/site-packages/airflow/providers/google/cloud/operators/dataproc.py", line 1899, in execute
    self.hook.wait_for_job(
  File "/home/tinmar/.local/lib/python3.10/site-packages/airflow/providers/google/common/hooks/base_google.py", line 462, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/tinmar/.local/lib/python3.10/site-packages/airflow/providers/google/cloud/hooks/dataproc.py", line 713, in wait_for_job
    raise AirflowException(f"Job failed:\n{job}")
airflow.exceptions.AirflowException: Job failed:
reference {
  project_id: "serene-gradient-371719"
  job_id: "b1b63d0a_subjob_avg_tincome"
}
placement {
  cluster_name: "***-spark-cluster"
  cluster_uuid: "b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca"
}
pyspark_job {
  main_python_file_uri: "gs://***_spark_bucket/par_task/avg_tincome.py"
  jar_file_uris: "gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
}
status {
  state: ERROR
  details: "Google Cloud Dataproc Agent reports job failure. If logs are available, they can be found at:\nhttps://console.cloud.google.com/dataproc/jobs/b1b63d0a_subjob_avg_tincome?project=serene-gradient-371719&region=us-central1\ngcloud dataproc jobs wait \'b1b63d0a_subjob_avg_tincome\' --region \'us-central1\' --project \'serene-gradient-371719\'\nhttps://console.cloud.google.com/storage/browser/***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/\ngs://***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/driveroutput"
  state_start_time {
    seconds: 1676301741
    nanos: 350156000
  }
}
status_history {
  state: PENDING
  state_start_time {
    seconds: 1676301714
    nanos: 571634000
  }
}
status_history {
  state: SETUP_DONE
  state_start_time {
    seconds: 1676301714
    nanos: 644830000
  }
}
status_history {
  state: RUNNING
  state_start_time {
    seconds: 1676301715
    nanos: 15779000
  }
}
driver_control_files_uri: "gs://***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/"
driver_output_resource_uri: "gs://***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/driveroutput"
labels {
  key: "***-version"
  value: "v2-3-0"
}
job_uuid: "ee6cc988-4536-4b4a-8d7b-caff8e07bebc"
done: true

[2023-02-13 09:22:27,255] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=7_dataproc_subdags.par_task, task_id=avg_tincome, execution_date=20230212T000000, start_date=20230213T152155, end_date=20230213T152227
[2023-02-13 09:22:27,265] {standard_task_runner.py:92} ERROR - Failed to execute job 83 for task avg_tincome (Job failed:
reference {
  project_id: "serene-gradient-371719"
  job_id: "b1b63d0a_subjob_avg_tincome"
}
placement {
  cluster_name: "***-spark-cluster"
  cluster_uuid: "b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca"
}
pyspark_job {
  main_python_file_uri: "gs://***_spark_bucket/par_task/avg_tincome.py"
  jar_file_uris: "gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
}
status {
  state: ERROR
  details: "Google Cloud Dataproc Agent reports job failure. If logs are available, they can be found at:\nhttps://console.cloud.google.com/dataproc/jobs/b1b63d0a_subjob_avg_tincome?project=serene-gradient-371719&region=us-central1\ngcloud dataproc jobs wait \'b1b63d0a_subjob_avg_tincome\' --region \'us-central1\' --project \'serene-gradient-371719\'\nhttps://console.cloud.google.com/storage/browser/***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/\ngs://***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/driveroutput"
  state_start_time {
    seconds: 1676301741
    nanos: 350156000
  }
}
status_history {
  state: PENDING
  state_start_time {
    seconds: 1676301714
    nanos: 571634000
  }
}
status_history {
  state: SETUP_DONE
  state_start_time {
    seconds: 1676301714
    nanos: 644830000
  }
}
status_history {
  state: RUNNING
  state_start_time {
    seconds: 1676301715
    nanos: 15779000
  }
}
driver_control_files_uri: "gs://***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/"
driver_output_resource_uri: "gs://***_spark_bucket/google-cloud-dataproc-metainfo/b7a28bbf-d1ca-4a77-b8f3-f482241cb5ca/jobs/b1b63d0a_subjob_avg_tincome/driveroutput"
labels {
  key: "***-version"
  value: "v2-3-0"
}
job_uuid: "ee6cc988-4536-4b4a-8d7b-caff8e07bebc"
done: true
; 3176)
[2023-02-13 09:22:27,293] {local_task_job.py:156} INFO - Task exited with return code 1
